redhat/frr.spec.in

```diff
@@ -106,16 +106,16 @@ Requires:       ncurses json-c
 Requires(pre):  /sbin/install-info
 Requires(preun): /sbin/install-info
 Requires(post): /sbin/install-info
-BuildRequires:  texi2html texinfo autoconf patch libcap-devel groff 
+BuildRequires:  texi2html texinfo autoconf automake patch libcap-devel groff
```

Comment:
I believe this, (and autoconf) are unnecessary, assuming what's put in SOURCES/ is still the dist tarball, which should have the output of autotools already, with no need to rerun them. The following is what I came up with after some testing in mock (the epel-7-x86_64 config). It appears to be sufficient (built against current master as of this comment), though some of it may be unnecessary:
```
BuildRequires:  texi2html texinfo patch libcap-devel groff
BuildRequires:  readline readline-devel ncurses ncurses-devel
BuildRequires:  json-c-devel bison flex make net-snmp-devel
BuildRequires:  libcap-devel c-ares-devel
```
plus the conditional `systemd` and `systemd-devel` you include below.
It should also be noted that texi2html (and the relevant init system, but mock already covers that) is needed just to parse the specfile, and must therefore be preinstalled.